x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformFailureHandler.java

```diff
@@ -234,13 +234,17 @@ private void handleIllegalArgumentException(IllegalArgumentException illegalArgu
      * @param numFailureRetries the number of configured retries
      */
     private void retry(Throwable unwrappedException, String message, boolean unattended, int numFailureRetries) {
+        // group failures to decide whether to report it below
```

Comment:
Is this change mostly for code quality? It doesn't change the logic right? I'm fine with moving it, just trying to understand that's all.